README.md

```diff
@@ -477,9 +477,13 @@ zcash-gitian$ vagrant plugin install --local
 ## Configure the version of zcash you want to build and sign
 
 Set the value of the `ZCASH_VERSION` variable in `.env` to point to the zcash commit you want to
-create a signature for. Likely you want the name of a git tag, such as `v2.0.5-2`.
-
+create a signature for. Likely you want the name of a git tag, usually the most recent released
+version.  You can determine this by running the following command in a freshly updated clone of the
+zcash repository:
```

Comment:
@garethtdavies @charlieok how's this?   Specifically _excluding_ `rc`s?

Reply:
Running that I get:
```
zcash$ git tag | grep -v rc |  tail -n 1
zc.v0.11.2.z0
```

We'll want the tags ordered by version number, and we'll want to exclude tags that don't match the pattern we're looking for. How about something like:

```
zcash$ git tag --sort=version:refname | grep ^v | grep -v rc |  tail -n 1
v2.0.6
```

Reply:
Huh!  What version of `git` are you using?

Mine:

```
git --version
git version 2.22.0
```

Reply:
$ git --version
git version 2.21.0 (Apple Git-120)

Seems unlikely to be due to a difference in git version. Does my command work on your system?

Reply:
This command works but seems a bit overkill. Why not link to Github tags page https://github.com/zcash/zcash/tags?

Reply:
Sounds like a good idea to me

Reply:
> ```
> $ git --version
> git version 2.21.0 (Apple Git-120)
> ```
> 
> Seems unlikely that these commands would work differently on our two versions of git. Is that what you're seeing? Does the command I suggested give the desired output on your setup?

I tested your command, and it gave the desired output.  I also agree that it seems implausible that `git tag`s interface would change (just wanted to rule it out).

Reply:
> This command works but seems a bit overkill. Why not link to Github tags page https://github.com/zcash/zcash/tags?

Acknowledged, I will make this change.